src/components/MemberButtons/index.js

```diff
@@ -5,37 +5,32 @@ import { Card } from 'antd';
 
 import './style.scss';
 
-class MemberButtons extends React.Component {
-  constructor(props) {
-    super(props);
+const MemberButtons = (props) => {
+  const {
```

Comment:
if a component doesnt need to be a class, ie is just receiving props and rendering something, this is simpler.